ui/push-health/Health.jsx

```diff
@@ -190,7 +187,9 @@ export default class Health extends React.PureComponent {
             rel="shortcut icon"
             href={result === 'fail' ? faviconBroken : faviconOk}
           />
-          <title>{`[${needInvestigationCount} failures] Push Health`}</title>
+          <title>{`[${
+            (status && status.testfailed) || 0
```

Comment:
This is a tangent, but I wonder if being able to rely on `testfailed` may be better than `needInvestigation` in my recent patch (https://github.com/mozilla/treeherder/pull/6998/files#diff-9036f1c80e93d7f3581d29707963ae23bf1427a82411009dd31e2de8913a113e) too. If so, I can update after this merges.

Reply:
That's a good point - 'needInvestigation' only refers to test failures, afaik. So if we are including lint and build failures here it'd make sense to do so consistently across the app. Especially since we'll be fixing the build and lint tabs in Push Health so it shows useful info, not just linking back to the jobs view.

Reply:
Cool! I'll wait for this to land and then rework my PR.